### DISCUSSION TITLE
Ideas for tech quiz questions

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [TechStudent10/techstudent10#13] [General]
I'm working on my tech quiz, it seems that your a guy who like me, watches Tech Youtubers, etc. Do you have any questions you would like to be added?

The temp branch for me working on it will be at - https://github.com/Kendall-Does-Coding-Websites/quiz/tree/workon-tech-quiz

**Comment by TechStudent10** [comment]
What does CPU stand for?
Answer: Central Processing Unit

What does RAM stand for?
Answer: Random Access Memory

Who created the Internet?
Answer: Tim Burners Lee

Who are the founders of Apple?
Answer: Steve Jobs and Steve Wozniak

When was Windows 3.1 released?
Answer: April 6, 1992

What Windows release had a Second Edition?
Answer: Windows 98

And the hardest one...

Who took the photo of the most iconic and most viewed image in the entire world... The default Windows XP wallpaper named Bliss?
Answer: Charles O'Rear

**Reply by KendallDoesCoding** [reply]
Interesting questions.. Will add a few of them when working on the tech quiz, but can you share the answers here too?

**Reply by TechStudent10** [reply]
Just added the answers.

**Reply by KendallDoesCoding** [reply]
Thanks

**Reply by KendallDoesCoding** [reply]
@TechStudent11 Hi,, I'm adding the questions to the topic now, can you give some options, 3 options + answer, so I can straight away add them?.... Thanks

**Reply by KendallDoesCoding** [reply]
Also, I don't think I'll had the 4th question.... I'll add most of the others.

**Reply by KendallDoesCoding** [reply]
> @TechStudent11 Hi,, I'm adding the questions to the topic now, can you give some options, 3 options + answer, so I can straight away add them?.... Thanks

I added CPU with some options, struggling to find options for the RAM question and the rest of the ones I'm working on adding soon, but idk what options I can add for the rest of them soon. can you give me options for the RAM question and everything else, except the CPU one? @TechStudent11

**Reply by TechStudent10** [reply]
For RAM, maybe:

Random Ape Making
IDK (yes, this is a possible answer)

and then seriously, idk.
